gemini/js-web-genkit/server.ts

```diff
@@ -0,0 +1,66 @@
+import { gemini15Flash, googleAI } from "@genkit-ai/googleai";
+import * as fs from 'fs/promises';
+import { genkit, z } from 'genkit';
+import { logger } from 'genkit/logging'
+import express from "express";
+
+const port = process.env.PORT || 3000;
+
+export const ai = genkit({
+    plugins: [googleAI()],
+});
+
+logger.setLogLevel('debug')
+const recipiePrompt = ai.definePrompt({
+        name: 'Recipies prompt',
```

Comment:
Maybe in the future there could be an option to use dotprompt files, or even just on the main genkit template

Reply:
Yeah, that's a good idea for a v2.